app/components/LeftPaneEntries/index.tsx

```diff
@@ -724,6 +913,47 @@ function LeftPane(props: Props) {
                         </>
                     </TabPanel>
                 )}
+                <TabPanel
+                    name="visuals"
+                    activeClassName={styles.visualsTab}
+                    retainMount="lazy"
+                >
+                    <Switch
+                        name="hide attachments"
+                        label="Hide Created entries"
+                        value={attachmentsWithEntriesHidden}
+                        onChange={setAttachmentsWithEntriesHidden}
+                    />
+                    <ListView
+                        spacing="comfortable"
+                        direction="vertical"
+                        data={leadPreviewAttachments}
+                        renderer={TablesAndVisualsItem}
+                        rendererParams={leadAttachmentItemRendererParams}
+                        className={styles.entryList}
+                        keySelector={leadAttachmentKeySelector}
+                        filtered={false}
+                        errored={false}
+                        pending={false}
+                        emptyIcon={(
+                            <Kraken
+                                variant="search"
+                            />
+                        )}
+                        emptyMessage="No entries found"
+                        messageShown
+                        messageIconShown
+                    />
+                    <Pager
+                        className={styles.pager}
+                        activePage={activePage}
+                        itemsCount={leadPreviewCount ?? 0}
```

Comment:
We should probably check for defined `leadPreviewCount` value and mount `Pager` conditionally